From 107e6ddf1c3cbb4a568a409cb6dbbe6469edc3c3 Mon Sep 17 00:00:00 2001
From: Ian Chien <ianchien.wiwynn@gmail.com>
Date: Fri, 2 Aug 2024 16:17:08 +0800
Subject: [PATCH 17/22] configurations: yosemite4: Rename the fan board config

Rename the fan board config with the led, Efuse, fan IC and ADC drivers.
Test case: Fan board config can be loaded and sensors can be loaded.

Change-Id: I374149b6011ded79180bf6e7d3ece1fa0ecba1e5
Signed-off-by: Ian Chien <ianchien.wiwynn@gmail.com>
---
 ... yosemite4_fanboard_fsc_max_adc_ti_led_nxp_efuse_mps.json} | 2 +-
 ...yosemite4_fanboard_fsc_nct_adc_max_led_ons_efuse_mps.json} | 2 +-
 meson.build                                                   | 4 ++--
 3 files changed, 4 insertions(+), 4 deletions(-)
 rename configurations/{yosemite4_fanboard_fsc_max_adc_ti.json => yosemite4_fanboard_fsc_max_adc_ti_led_nxp_efuse_mps.json} (99%)
 rename configurations/{yosemite4_fanboard_fsc_nct_adc_max.json => yosemite4_fanboard_fsc_nct_adc_max_led_ons_efuse_mps.json} (99%)

diff --git a/configurations/yosemite4_fanboard_fsc_max_adc_ti.json b/configurations/yosemite4_fanboard_fsc_max_adc_ti_led_nxp_efuse_mps.json
similarity index 99%
rename from configurations/yosemite4_fanboard_fsc_max_adc_ti.json
rename to configurations/yosemite4_fanboard_fsc_max_adc_ti_led_nxp_efuse_mps.json
index c8f8ff0..8d792dc 100644
--- a/configurations/yosemite4_fanboard_fsc_max_adc_ti.json
+++ b/configurations/yosemite4_fanboard_fsc_max_adc_ti_led_nxp_efuse_mps.json
@@ -684,7 +684,7 @@
         }
     ],
     "Name": "Yosemite 4 Fan Board $bus % 30",
-    "Probe": "xyz.openbmc_project.FruDevice({'BOARD_PRODUCT_NAME': 'FAN Board FSC-MAX ADC-TI', 'PRODUCT_PRODUCT_NAME': 'Yosemite V4'})",
+    "Probe": "xyz.openbmc_project.FruDevice({'BOARD_PRODUCT_NAME': 'Fan Board FSC-MAX ADC-TI LED-NXP EFUSE-MPS', 'PRODUCT_PRODUCT_NAME': 'Yosemite V4'})",
     "Type": "Board",
     "xyz.openbmc_project.Inventory.Decorator.Asset": {
         "Manufacturer": "$PRODUCT_MANUFACTURER",
diff --git a/configurations/yosemite4_fanboard_fsc_nct_adc_max.json b/configurations/yosemite4_fanboard_fsc_nct_adc_max_led_ons_efuse_mps.json
similarity index 99%
rename from configurations/yosemite4_fanboard_fsc_nct_adc_max.json
rename to configurations/yosemite4_fanboard_fsc_nct_adc_max_led_ons_efuse_mps.json
index 4b8bf23..6aaaa42 100644
--- a/configurations/yosemite4_fanboard_fsc_nct_adc_max.json
+++ b/configurations/yosemite4_fanboard_fsc_nct_adc_max_led_ons_efuse_mps.json
@@ -684,7 +684,7 @@
         }
     ],
     "Name": "Yosemite 4 Fan Board $bus % 30",
-    "Probe": "xyz.openbmc_project.FruDevice({'BOARD_PRODUCT_NAME': 'FAN Board FSC-NCT ADC-MAX', 'PRODUCT_PRODUCT_NAME': 'Yosemite V4'})",
+    "Probe": "xyz.openbmc_project.FruDevice({'BOARD_PRODUCT_NAME': 'Fan Board FSC-NCT ADC-MAX LED-ONS EFUSE-MPS', 'PRODUCT_PRODUCT_NAME': 'Yosemite V4'})",
     "Type": "Board",
     "xyz.openbmc_project.Inventory.Decorator.Asset": {
         "BuildDate": "$BOARD_MANUFACTURE_DATE",
diff --git a/meson.build b/meson.build
index 83ae345..30c101c 100644
--- a/meson.build
+++ b/meson.build
@@ -193,8 +193,8 @@ configs = [
     'vegman_rx20_baseboard.json',
     'vegman_sx20_baseboard.json',
     'wft_baseboard.json',
-    'yosemite4_fanboard_fsc_max_adc_ti.json',
-    'yosemite4_fanboard_fsc_nct_adc_max.json',
+    'yosemite4_fanboard_fsc_max_adc_ti_led_nxp_efuse_mps.json',
+    'yosemite4_fanboard_fsc_nct_adc_max_led_ons_efuse_mps.json',
     'yosemite4_sentineldome_t1.json',
     'yosemite4_sentineldome_t1_retimer.json',
     'yosemite4_sentineldome_t2.json',
-- 
2.25.1

